docker.go

```diff
@@ -432,13 +434,25 @@ func (d *docker) stopContainer(ctx context.Context, id string) error {
 	stopTimeout := defaultStopTimeout
 
 	err := d.client.ContainerStop(ctx, id, &stopTimeout)
-	if err != nil {
+	if err != nil && !client.IsErrNotFound(err) {
 		return fmt.Errorf("can't stop container %s: %w", id, err)
 	}
 
 	return nil
 }
 
+func (d *docker) removeContainer(ctx context.Context, id string) error {
+	d.lock.Lock()
+	defer d.lock.Unlock()
+
+	err := d.client.ContainerRemove(ctx, id, types.ContainerRemoveOptions{Force: true})
+	if err != nil && !client.IsErrNotFound(err) && isDeletionAlreadyInProgessError(err, id) {
```

Comment:
I think `isDeletionAlreadyInProgessError` should be negated just like "not found" error, we want to ignore it instead of failing container removal.

Reply:
Thanks for the hint! I had to change the logic of `isDeletionAlreadyInProgessError`. It seems like it was not really checking for the error. Now I was able to get it right and negate it.